### PR TITLE
Do not uppercase table headers

### PR DESCRIPTION
**Pull request by snowystinger**
Closes 

One thing to note, on overflow wrap tables, the headers have now gotten taller by 4px because we start with an estimated height and then calculate after render the actual height.

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

**Review by reidbarber — DISMISSED** [review]
LGTM

**Comment by devongovett** [comment]
Looking at https://spectrum.adobe.com/page/table/, I think the font size looks too big, and the color looks too dim.

File: packages/@adobe/spectrum-css-temp/components/table/skin.css [hunk 1]
```diff
@@ -16,19 +16,20 @@ governing permissions and limitations under the License.
 }
 
 .spectrum-Table-headCell {
-  color: var(--spectrum-table-header-text-color);
+  color: var(--spectrum-alias-text-color);
```
Comment (LFDanLu) [inline packages/@adobe/spectrum-css-temp/components/table/skin.css:19]: The text color in the XD file is #000000/rgb(0,0,0), but this var resolves to rgb(75,75,75) on light. Do we need to adjust the value of the var? Or is it just that we need to update our spectrum vars?

**Review by LFDanLu — COMMENTED** [review]
Partial review, need to track down the other component XD files.

File: packages/@adobe/spectrum-css-temp/components/table/skin.css (hunk 1 above)
Comment (snowystinger) [reply packages/@adobe/spectrum-css-temp/components/table/skin.css:19]: update our spectrum vars
for any reviewers who don't already know, you can get the spectrum plugin for XD, then when you click on a component, it'll tell you the spectrum variable names. you can swap between version 5 and 6

File: packages/@adobe/spectrum-css-temp/components/accordion/index.css [hunk 2]
```diff
@@ -67,16 +66,14 @@ governing permissions and limitations under the License.
...
   padding-inline-end: var(--spectrum-accordion-item-padding);
   margin: 0;
 
   min-height: var(--spectrum-accordion-item-height-actual);
 
-  font-size: var(--spectrum-accordion-item-title-text-size);
+  font-size: var(--spectrum-global-dimension-font-size-100);
   line-height: var(--spectrum-accordion-item-line-height);
-  text-transform: uppercase;
-  letter-spacing: var(--spectrum-accordion-item-title-tracking-actual);
```
Comment (devongovett) [inline packages/@adobe/spectrum-css-temp/components/accordion/index.css:77]: These don't look like the xd file at all, but I guess we don't care for now? Will it be weird if people are using the alpha though?

File: packages/dev/docs/src/docs.css [hunk 3]
```diff
@@ -747,12 +747,11 @@ h2.sectionHeader {
 
   .propTable td[data-column]:before {
     content: attr(data-column);
-    font-size: var(--spectrum-table-header-text-size);
+    font-size: var(--spectrum-global-dimension-font-size-100);
     font-weight: var(--spectrum-table-header-text-font-weight);
     min-height: var(--spectrum-table-header-min-height);
     letter-spacing: var(--spectrum-table-header-text-letter-spacing);
```
Comment (devongovett) [inline packages/dev/docs/src/docs.css:753]: Remove this one too.

File: packages/dev/docs/src/docs.css [hunk 4]
```diff
@@ -747,12 +747,11 @@ h2.sectionHeader {
...
     color: var(--spectrum-table-header-text-color);
```
Comment (devongovett) [inline packages/dev/docs/src/docs.css:754]: wrong color.

File: packages/@adobe/spectrum-css-temp/components/accordion/index.css (hunk 2 above)
Comment (snowystinger) [reply packages/@adobe/spectrum-css-temp/components/accordion/index.css:77]: yeah, agreed, i've updated to be closer, including fixing our top border

**Review by devongovett — APPROVED** [review]

**Comment by ktabors** [comment]
Where are the accordion XD files? I'm sure the larger and darker side is desired it just looks weird. I wanted to check the chevron because the open state seems too low. Probably not a priority since this is alpha and main work is being done by Quarry.

File: packages/@adobe/spectrum-css-temp/components/table/skin.css (hunk 1 above)
Comment (ktabors) [reply packages/@adobe/spectrum-css-temp/components/table/skin.css:19]: Is this the same issue I'm seeing where the sort direction indicator looks darker in the XD file and v5 colors says it should be #6d6d6, closest Spectrum being gray-700, but we're using #8e8e8e/rgb(142,142,142). When I look at the spectrum token of that color we're using gray-600, which isn't gray-700.

File: packages/@adobe/spectrum-css-temp/components/table/skin.css [hunk 5]
```diff
@@ -16,19 +16,20 @@ governing permissions and limitations under the License.
...
       color: var(--spectrum-table-header-sort-icon-color);
     }
 
     &.is-hovered {
-      color: var(--spectrum-table-header-text-color-hover);
+      color: var(--spectrum-alias-text-color-hover);
 
       .spectrum-Table-sortedIcon {
-        color: var(--spectrum-table-header-sort-icon-color-hover);
+        color: var(--spectrum-alias-text-color-hover);
```
Comment (ktabors) [inline packages/@adobe/spectrum-css-temp/components/table/skin.css:32]: see above comment, I'm not sure this is correct.

**Comment by snowystinger** [comment]
> Where are the accordion XD files? I'm sure the larger and darker side is desired it just looks weird. I wanted to check the chevron because the open state seems too low. Probably not a priority since this is alpha and main work is being done by Quarry.

accordion xd are in the contributions site

> Is this the same issue I'm seeing where the sort direction indicator looks darker in the XD file and v5 colors says it should be #6d6d6, closest Spectrum being gray-700, but we're using #8e8e8e/rgb(142,142,142). When I look at the spectrum token of that color we're using gray-600, which isn't gray-700.

Sort icon was wrong, should be gray-800 like the text

**Review by ktabors — APPROVED** [review]

**Review by reidbarber — APPROVED** [review]

**Pull request merged**